app/models/manageiq/providers/kubevirt/infra_manager/vm.rb

```diff
@@ -49,41 +49,6 @@ def queue_generate_remote_viewer_file
     MiqTask.generic_action_with_callback(task_options, queue_options)
   end
 
-  #
-  # This method is executed in the provider worker. It contacts the KubeVirt API to get the details of the connection to
-  # the console, and generates the `remote-viewer` configuration file.
-  #
-  # @return [String] The content of the cofiguration file.
-  #
-  def generate_remote_viewer_file
```

Comment:
As far as I can tell this was never called

Reply:
Should this be killed also, then?

https://github.com/ManageIQ/manageiq-providers-kubevirt/blob/4cb507a2d8bf2bca505d3bcdf6715aca70833fa9/app/models/manageiq/providers/kubevirt/infra_manager/vm.rb#L31

Reply:
I also found this semi-related method in ovirt (and rhv):

https://github.com/ManageIQ/manageiq-providers-ovirt/blob/8c8a8e3d01c2b1e766bc690a076e6cca22416d1b/app/models/manageiq/providers/ovirt/infra_manager/ovirt_services.rb#L205

Reply:
> Should this be killed also, then?

Yes, done

> I also found this semi-related method in ovirt (and rhv):

Ovirt/RHV still use spice so that is still used, spice was only removed from kubevirt

Reply:
Weird - the comment on the queue method says ui-classic calls it - but I checked the entire diff history of ui-classic and it's not there.

Reply:
Yeah I checked the same, my guess is that it was started but never finished